app/assets/stylesheets/articles.scss

```diff
@@ -4,301 +4,73 @@
 @import 'variables';
 @import '_mixins';
 
+
 .home {
   position: relative;
-  min-height: 440px;
-  margin: auto;
+  min-height: 440px; // TODO: why?
+  margin: 0 auto;
```

Comment:
There are some styles in `_top.html.erb` which can be removed. They were added due to a weird css specificity bug that only occurred in production so we quickly hacked it in.

It can be removed and if we happen to encounter the same bug again we can fix it again. 🤷‍♂

Reply:
Since this PR is (and will) not be about  actual CSS refactoring primarily, I'd rather keep it as it is for now and make sure it's not a _breaking change_ in some other PR. That `// TODO: ...` was more of a note to myself to check it in future.